### ISSUE TITLE
Freeze when initializing Bugsnag

### ISSUE THREAD

**Issue by vanniktech** [bugsnag/bugsnag-java#176]
The SQLDelight IntelliJ Idea plugin uses Bugsnag. When opening the settings from IntelliJ, bug snag gets initialized via sqldelight which blocks the UI.

```
"AWT-EventQueue-0" prio=0 tid=0x0 nid=0x0 runnable
     java.lang.Thread.State: RUNNABLE
 (in native)
	at java.base@11.0.13/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.base@11.0.13/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(InetAddress.java:929)
	at java.base@11.0.13/java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1519)
	at java.base@11.0.13/java.net.InetAddress$NameServiceAddresses.get(InetAddress.java:848)
	at java.base@11.0.13/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
	at java.base@11.0.13/java.net.InetAddress.getLocalHost(InetAddress.java:1641)
	at com.bugsnag.callbacks.DeviceCallback.lookupHostname(DeviceCallback.java:44)
	at com.bugsnag.callbacks.DeviceCallback.getHostnameValue(DeviceCallback.java:22)
	at com.bugsnag.callbacks.DeviceCallback.initializeCache(DeviceCallback.java:52)
	at com.bugsnag.Configuration.<init>(Configuration.java:58)
	at com.bugsnag.Bugsnag.<init>(Bugsnag.java:92)
	at app.cash.sqldelight.intellij.SqlDelightErrorHandler.<init>(SqlDelightErrorHandler.kt:32)
...
```

There's also a thread dump:

[threadDumps-freeze-20220512-111722-AI-213.7172.25.2211.8563311-5sec.zip](https://github.com/bugsnag/bugsnag-java/files/8702300/threadDumps-freeze-20220512-111722-AI-213.7172.25.2211.8563311-5sec.zip)

I've reported it to both [Android Studio](https://issuetracker.google.com/issues/232348779) as well as [sqldeilght](https://github.com/cashapp/sqldelight/issues/3230). 

There also seem to be some suggested workarounds:

- https://stackoverflow.com/questions/33289695/inetaddress-getlocalhost-slow-to-run-30-seconds
- https://github.com/gradle/gradle/issues/11122

**Comment by luke-belton** [comment]
Hi @vanniktech thanks for raising this. We're going to investigate and see what we can do to address this in the `bugsnag-java` library. Note that any fixes would also require the SQLDelight plugin maintainers to bump the version of Bugsnag they're using.

**Comment by vanniktech** [comment]
Hey @luke-belton thanks. I'm sure that they will gladly bump the version

**Comment by luke-belton** [comment]
Hi @vanniktech - just to let you know that we [added a fix for the issue you reported above](https://github.com/bugsnag/bugsnag-java/pull/179), which was released in [`v3.6.4`](https://github.com/bugsnag/bugsnag-java/releases/tag/v3.6.4) of `bugsnag-java` 🎉
